frontend/www/js/omegaup/time.ts

```diff
@@ -255,13 +259,46 @@ export function remoteTimeAdapter<T>(value: T): T {
  * Calculate the duration of a contest based on its start date and its end date.
  * @param startDate - The start date of a contest
  * @param finishDate - The finish date of a contest
- * @returns The duration of a contest in human redeable format (HH:mm:ss)
+ * @returns The duration of a contest in human readable format (Locale based)
  */
 export function formatContestDuration(
   startDate: Date,
   finishDate: Date,
 ): string {
-  return formatDelta(finishDate.getTime() - startDate.getTime());
+  let delta = finishDate.getTime() - startDate.getTime();
+  const days = Math.floor(delta / (24 * 60 * 60 * 1000));
+  delta -= days * (24 * 60 * 60 * 1000);
+  const hours = Math.floor(delta / (60 * 60 * 1000));
+  delta -= hours * (60 * 60 * 1000);
+  const minutes = Math.floor(delta / (60 * 1000));
+  delta -= minutes * (60 * 1000);
+  const seconds = Math.floor(delta / 1000);
+  let currentLocale;
+  switch (T.locale) {
+    case 'es':
+      currentLocale = esLocale;
+      break;
+    case 'pt':
+      currentLocale = ptLocale;
+      break;
+    case 'en':
+      currentLocale = enLocale;
+      break;
+    default:
+      currentLocale = enLocale;
```

Comment:
```suggestion
    case 'es':
    default:
      currentLocale = esLocale;
```

en toda la página usamos español como default.